tests/dir.props

```diff
@@ -106,9 +118,10 @@
     <DnuRestoreSource>@(DnuSourceList -> '--source %(Identity)', ' ')</DnuRestoreSource>
     <DnuRestoreDirs>@(DnuRestoreDir -> '%(Identity)', ' ')</DnuRestoreDirs>
 
-    <DnuRestoreCommand>$(DnuRestoreCommand) "$(DotnetToolCommand)"</DnuRestoreCommand>
+    <DnuRestoreCommand>$(DnuRestoreCommand) $(DotnetToolCommand)</DnuRestoreCommand>
```

Comment:
@rartemev You are correct.  However, cross build support is less than ideal now, and this change may enable substantial improvement to them.  You may or may not be assigned that task.